README.md

```diff
@@ -1 +1,74 @@
-# BookStore
\ No newline at end of file
+# BookStore | M3Wx
+
+> "BookStore" is a web app for adding and removing books from a library storage. It is a Single Page App (SPA) that allows users to add and remove a book
+
+![screenshot](branding/brand-demo.png)
+
+Our goal here is to Build a single page web application for book weebs using the React library
+
+## Live Demo
+Click [here](https://harlexkhal.github.io/BookStore/) to view live demo
+
+### Features implemented so far are:
+
+- Project Setup with React Installations
+- Calculator with (Add, Minus, Multiply, and Division) Features
+- Page for Mathematics Quotes
+- Home page that gives information about the history of mathematics
```

Comment:
- [ ] Please kindly update this select to reflect the implemented project, this section is talking about the `React calculator` instead of `Bookstore`. Please update it, as this will make your readme file more informative and professional.

Reply:
Yeah Thanks, Completely forgot about that, thanks for pointing that out 🙇‍♂️